src/wad/tex.rs

```diff
@@ -230,13 +241,15 @@ impl TextureDirectory {
         let atlas_size = atlas_size;
 
         assert!(positions.len() == entries.len());
-        // TODO(cristicbz): This should probably split things into multiple atlases or something,
+        // TODO(cristicbz): This should probably split things into multiple atlases or
+        // something,
         // but realistically, I'm never going to implement that.
```

Comment:
Could you please concatenate with the the previous line. The same comment reflow thing.